en/non-node.md

```diff
@@ -34,7 +34,7 @@ var vm = new Vue({
   }
```

Comment:
Just by curiosity. I imagine, one of goal is to provide a `'<div data-server-rendered="true">...</div>'` render in this case so, because this will be asynchrone, how use the result chain ? It's `print(res)` will do the Job but stop the PHP process ?

So in this case 

```
<?
...
 $v8->executeString($vue_source);
 $v8->executeString($renderer_source);
 $v8->executeString($app_source);
echo "test";
 ?>
```
"test" will be output after (and block next part of code) or before render? I think the first because a PHP script is executed synchronously, right ?

Reply:
I would answer if I knew the answer. 😅